contrib/guix/patches/glibc-2.27-riscv64-fix-incorrect-jal-with-HIDDEN_JUMPTARGET.patch

```diff
@@ -0,0 +1,41 @@
+Backported from: https://sourceware.org/git/?p=glibc.git;a=commit;h=68389203832ab39dd0dbaabbc4059e7fff51c29b
+Context: https://sourceware.org/bugzilla/show_bug.cgi?id=28509
+
+Resolves a build failure with glibc 2.27 + binutils >=2.40.
+Patch can be removed if we update glibc to >= 2.35.
```

Comment:
The post notes glibc 2.39, is this patch still needed?

Reply:
Yes. We build a custom cross-compilation toolchain that uses [glibc 2.27](https://github.com/monero-project/monero/blob/master/contrib/guix/manifest.scm#L181) instead of 2.39. This patch fixes that build.

We use a lower version of glibc to build Monero than what other Guix packages are linked against. This is because the glibc version we link against determines the minimum glibc version that must be available at _runtime_. A binary linked against glibc 2.39 will run on Ubuntu 22.04 (glibc 2.39), but isn't guaranteed to run on Ubuntu 22.04 (glibc 2.35). Ubuntu 18.04 is the lowest version we can support by using glibc 2.27.